### DISCUSSION TITLE
Option to disable accuracy pop up in break time

### DISCUSSION THREAD

**Discussion by MasterOfDarkness727** [ppy/osu#30529] [Ideas]
I would really like an option to disable the accuracy display in the middle of the screen in break time. When you are FCing a long or difficult map seeing 99 or 100% accuracy in the middle of the screen really doesn't help with nerves. I think it would be really helpful if you could change what is displayed in break time or an option to turn the accuracy display off.
![osu_2024-11-06_23-26-22](https://github.com/user-attachments/assets/a58c93d6-c5f1-43b0-b5fe-dccdc7ea7e00)

**Comment by peppy** [comment]
An eventual goal would be to have the break screen completely skinnable so you can customise what it does and doesn't display. I don't believe we will be adding customisation until this can happen.

**Reply by gnud-gnaoh** [reply]
How about just don't display the accuracy and grade entirely? Those who want it will already have it on their skin already. And for those who doesn't, this impacts nerves and gameplay quite a lot.

**Reply by MasterOfDarkness727** [reply]
"once you add a feature you can never remove it" said peppy himself after he removed the song select background blur and people started complaining,  so that's definitely not happening
